pkg/client/cmd/root.go

```
@@ -57,21 +58,27 @@ func NewKamelCommand(ctx context.Context) (*cobra.Command, error) {
 	cmd.PersistentFlags().StringVarP(&options.Namespace, "namespace", "n", "", "Namespace to use for all operations")
 
 	// Parse the flags before setting the defaults
-	cmd.ParseFlags(os.Args)
+	err := cmd.ParseFlags(os.Args)
```

after digging a little bit more this one is also failing, don't know why btw

---

i guess we should ignore the error for the moment